setup.py

```diff
@@ -29,8 +27,6 @@
           'Topic :: Multimedia :: Graphics :: 3D Rendering',
           'Topic :: Scientific/Engineering :: Visualization',
     ],
-    install_requires=['jupyter', 'vpython'],
-    #cmdclass=cmdclass('vpnotebook/data'),
-    #package_data={'vpython': ['data/*'], 'vpnotebook': ['data/kernel.json']},
+    install_requires=['jupyter', 'vpnotebook'],
```

Comment:
I believe that dependency has been removed, or at least I think that was the intent. Yes, I just checked and it seems to be gone now.